<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToServer</class>
 <widget class="QWidget" name="ConnectToServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="localAddress_checkBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>connect to local host server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectToServer_pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>42</y>
     <width>363</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Your Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ip_label">
        <property name="text">
         <string>IP Address :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_lineEdit1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_lineEdit2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_lineEdit3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_lineEdit4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>ip_lineEdit1</tabstop>
  <tabstop>ip_lineEdit2</tabstop>
  <tabstop>ip_lineEdit3</tabstop>
  <tabstop>ip_lineEdit4</tabstop>
  <tabstop>localAddress_checkBox</tabstop>
  <tabstop>connectToServer_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
